src/mixer/basetrackplayer.cpp

```diff
@@ -527,7 +527,6 @@ void BaseTrackPlayerImpl::slotTrackLoaded(TrackPointer pNewTrack,
         m_pDuration->set(0);
         m_pFileBPM->set(0);
         m_pKey->set(0);
-        setReplayGain(0);
```

Comment:
why was this the wrong place to put the replaygain setting?

Reply:
Because it was too late, the first engine call uses the replay gain of the previous loaded track.